DOCSP-30943: add bson lib explicitly (#106)
mongodb/docs-kotlin: source/whats-new.txt

What's New in 4.10
------------------

.. important::

   Starting in version 4.10.1 of the {+driver-short+}, you must add

>    the ``bson-kotlinx`` library as an explicit dependency for using the 

Reviewer: ```suggestion
   the ``bson-kotlinx`` library as an explicit dependency to use the 
```
